packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -0,0 +1,50 @@
+import { Component, Fragment, createElement } from 'react';
+import PropTypes from 'prop-types';
+
+import classify from 'src/classify';
+import defaultClasses from './product.css';
+
+class Product extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
```

Comment:
This is missing some classes used in `render` and `get options`

Reply:
Added missing prop-types for this and others. 👍